vars/buildAndProvideDebianPackage.groovy

```diff
@@ -9,6 +9,9 @@ def call(Boolean isArchIndependent = false, List ignoredArchs = []) {
       // Only 'Build source' stage requires checkout.
       skipDefaultCheckout()
     }
+    environment {
+        TELEGRAM_BOT_CREDS = credentials('a25d8b20-4a81-43e9-ac37-dcfb5285790a')
+    }
```

Comment:
We don't want this environment variable to be available to the entire build, only to the specific `post {}` section that uses it. 

Note that you can test your changes by adding your branch to the `@Library` definition in a Jenkinsfile in the org, for example `@Library('ubports-build-tools@Flohack74-patch-1') _`

Reply:
Humm I am not sure it can be defined in post? At least I did only find it for stage, or the whole job in the docs. But I can try...